ext/parser/bq_parser_test.go

```diff
@@ -99,7 +99,6 @@ func TestParseTopLevelUpstreamsFromQuery(t *testing.T) {
 				ExpectedResourceURNs: []bigquery.ResourceURN{
 					newBQResourceURN("project", "fire", "fly"),
 					newBQResourceURN("project", "maximum", "overdrive"),
-					newBQResourceURN("project", "maximum", "overdrive"), // the result should be unique(?))
```

Comment:
why we needed to remove this ?

Reply:
it should not yield duplicate upstream, logic on transformer returns unique resource https://github.com/goto/transformers/blob/aeee206d8e2976d0e3ee55aacf87f419997a4009/task/bq2bq/upstream/parser.go#L84C24-L84C38